tests/bundle/Functional/TestCase.php

```diff
@@ -87,7 +87,7 @@ protected function setUp(): void
     {
         parent::setUp();
 
-        $this->httpHost = getenv('EZP_TEST_REST_HOST') ?: 'localhost';
+        $this->httpHost = getenv('EZP_TEST_REST_HOST') ?: '127.0.0.1:8000';
```

Comment:
Looks like something that depends on your local environment. Should be left to `localhost`.